src/status_im/contexts/settings/wallet/keypairs_and_accounts/view.cljs

```diff
@@ -0,0 +1,104 @@
+(ns status-im.contexts.settings.wallet.keypairs-and-accounts.view
+  (:require [clojure.string :as string]
+            [quo.core :as quo]
+            [quo.theme]
+            [react-native.core :as rn]
+            [react-native.safe-area :as safe-area]
+            [status-im.constants :as constants]
+            [status-im.contexts.settings.wallet.keypairs-and-accounts.actions.view :as actions]
+            [status-im.contexts.settings.wallet.keypairs-and-accounts.style :as style]
+            [utils.address :as utils]
+            [utils.i18n :as i18n]
+            [utils.re-frame :as rf]))
+
+(defn navigate-back
+  []
+  (rf/dispatch [:navigate-back]))
+
+(defn- parse-accounts
+  [given-accounts]
+  (->> given-accounts
+       (filter (fn [{:keys [path]}]
+                 (not (string/starts-with? path constants/path-eip1581))))
+       (map (fn [{:keys [customization-color emoji name address]}]
+              {:account-props {:customization-color customization-color
+                               :size                32
+                               :emoji               emoji
+                               :type                :default
+                               :name                name
+                               :address             address}
+               :networks      []
+               :state         :default
+               :action        :none}))))
+
+(defn on-options-press
+  [{:keys [theme]
+    :as   props}]
+  (rf/dispatch [:show-bottom-sheet
+                {:content (fn [] [actions/view props])
+                 :theme   theme}]))
+
+(defn- keypair
+  [item index _
+   {:keys [profile-picture compressed-key customization-color]}]
+  (let [theme            (quo.theme/use-theme)
+        accounts         (parse-accounts (:accounts item))
+        default-keypair? (zero? index)
+        details          {:full-name (:name item)
+                          :address   (when default-keypair?
+                                       (utils/get-shortened-compressed-key compressed-key))}
+        on-press         (rn/use-callback
+                          (fn []
+                            (on-options-press
+                             (merge {:theme theme
+                                     :title (:full-name details)}
+                                    (if default-keypair?
+                                      {:type                :default-keypair
+                                       :description         (:address details)
+                                       :customization-color customization-color
+                                       :profile-picture     profile-picture}
+                                      {:type        :keypair
+                                       :icon-avatar :i/seed}))))
+                          [customization-color default-keypair? details profile-picture theme])]
```

Comment:
* Here we're passing along the props for the bottom-sheet that will open after pressing the options button for a key-pair component.  
* The `drawer-top` is used in the bottom-sheet view, so we need to pass along different props depending on whether we're opening the actions menu for the default key-pair or another key-pair.